microovn/api/certificates/issue.go

```diff
@@ -52,7 +52,7 @@ func issueCertificatesPut(s state.State, r *http.Request) response.Response {
 	// Fail with 404 if requested service is not enabled
 	if !isCertificateAllowed {
 		missingMsg := fmt.Sprintf(
-			"Can't issue certificate for service '%s'. Service is not enabled on this member. Enabled services: %s",
+			"can't issue certificate for service '%s'. service is not enabled on this member. enabled services: %s",
```

Comment:
nit: This error messages consists of multiple sentences, does the linter complain about the capitalization of "inner" sentences as well? If yes, perhaps we should replace dots with commas, to keep it a single sentence.